pkg/kgo/client.go

```diff
@@ -454,7 +454,6 @@ func NewClient(opts ...Opt) (*Client, error) {
 	}
 
 	ctx, cancel := context.WithCancel(context.Background())
-
```

Comment:
in the interest of keeping conflicts to minimum with upstream, we probably want this new line 😄

Reply:
_addressed @ 47adbd15302565bc4e2c3353949f22304bbd1aa9_